src/writer/json/json_writer.py

```diff
@@ -0,0 +1,26 @@
+#  Copyright (c) 2022 spdx contributors
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import json
+
+from src.jsonschema.document_converter import DocumentConverter
+from src.model.document import Document
+
+
+class JsonWriter:
```

Comment:
I don't think a class is necessary here as there is no state to it

Reply:
Yeah, alright. Converted to a function, with the option of passing a converter (so we don't have to create a new instance each time in case it is called multiple times, the user could pass the same instance)